src/shared/components/palette/collection/write-collection-field.html

```diff
@@ -5,7 +5,7 @@
     <h2 class="heading-h2 error-spacing">
       {{caseField | ccdFieldLabel}}
     </h2>
-    <button class="button" type="button" (click)="addItem(true)" [disabled]="isNotAuthorisedToCreate()">Add new</button>
+    <button class="button" type="button" (click)="addItem(true)" [disabled]="isNotAuthorisedToCreate() || isSearchFilter()">Add new</button>
```

Comment:
_Nitpick:_
Really shouldn't use function bindings from a performance point of view. This now has _two_ function bindings. 😬